media/js/base/search-activist.js

```diff
@@ -0,0 +1,57 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function($) {
+    'use strict';
+
+    var $searchBar = $('#moz-search');
+    var terms = $searchBar.data('terms').split(',');
+    var displayTerm = $('.search-label-term');
+
+    function changeTerm() {
+        var term = displayTerm.data('term') || 0;
+        displayTerm.data('term', term === terms.length -1 ? 0 : term + 1).text(terms[term]).fadeIn(150).delay(3500).fadeOut(150, changeTerm);
+    }
+    changeTerm();
+
+    displayTerm.on('click', function() {
+        $('input.gsc-input').val(displayTerm.text());
```

Comment:
Nit - should cache `$('input.gsc-input')` so jQuery doesn't have to look it up each time.